### PR TITLE
BugID:1761:Add the source code of weight scale.

### PR DESCRIPTION
**Pull request by wzhiyuan-nit**
[Detail]
Issue description:
#1761 
Rootcause:

Solution:

Requirement description:

Solution:

[Verified Cases]
Build Pass: <solution list>
Test Pass: <test case list>

Signed-off-by: wzhiyuan-nit <wuming.wzy@alibaba-inc.com>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 3c47d13590a7487d1098dabdbb77b157a30d30a5

**Comment by skylarCai** [comment]
auto build success,number:211

**Comment by skylarCai** [comment]
autotest fail,number:1564

**Review by ethanlcz — APPROVED** [review]

File: haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/README.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+&emsp;&emsp;
+将[全部代码](./code/)文件复制并覆盖刚刚创建的工程目录下的原有文件，之后对代码进行如下修改。
+
+1. **填写Wi-Fi名称及密码**
+
+&emsp;&emsp;
+在main.py中，填写可用的Wi-Fi名称及密码。
+
+``` python
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/README.md:161]: LGFM

File: haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/README.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+wifi_password = "请填写您的路由器密码"
+```
+
+2. **修改设备端三元组**
+
+&emsp;&emsp;
+在main.py中，填写创建的设备三元组信息。关于设备三元组的获取，请参考[创建设备及获取三元组](./README.md "创建设备及获取三元组")中的步骤。
+
+``` python
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/README.md:173]: LGFM

Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/README.md:173]: LGFM

File: haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/code/main.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+# 设备实例
+clkDev = None
+dataDev = None
+dcDev = None
+resDev = None
+spi1Dev = None
+
+# 三元组信息
+productKey = "产品key"
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/code/main.py:25]: LGFM

Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/weight_scale/haaseduk1/code/main.py:25]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**